tests/cpp/test_multidevice_sharding.cpp

```diff
@@ -951,86 +930,6 @@ TEST_F(MultiDeviceTest, LoopShardedMergeReshapeIds) {
       __FILE__);
 }
 
-TEST_F(MultiDeviceTest, TransposeSchedulerWithView) {
```

Comment:
Removing this test, since such a view should not trigger transpose scheduler now.